examples/src/app/equal-to-validator/equal-to-validator.component.ts

```diff
@@ -0,0 +1,33 @@
+import { Component, OnDestroy, OnInit } from '@angular/core';
+import { email, Validator } from '../items';
+import { ActivatedRoute } from '@angular/router';
+
+@Component({
+  selector: 'app-equals-to-field-validator',
+  templateUrl: './equal-to-validator.component.html'
+})
+export class EqualToValidatorComponent implements OnInit, OnDestroy {
+
+  private sub: any;
+  emailItems = email;
```

Comment:
You need to import `equalTo` instead `email`. Because at the moment the code examples on the website are the wrong

Reply:
sorry, my mistake